content/intro-to-storybook/vue/fr/deploy.md

```diff
@@ -0,0 +1,99 @@
+
+### GitHub
+
+Si vous suivez le chapitre de test précédent, passez à la fconfiguration d'un référentiel sur GitHub.
+
+Lorsque le projet a été initialisée avec le Vue CLI, un référentiel local était déjà configuré pour vous. A ce stade, il est sûr d'ajouter les fichiers au premier commit.
+
+```bash
+$ git add .
+```
+
+Maintenant, commitez les fichiers.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Configurez un référentiel GitHub
+
+Allez sur GitHub et configurez un référentiel [ici](https://github.com/new). Nommez votre repo “taskbox”.
+
+![GitHub setup](/intro-to-storybook/github-create-taskbox.png)
+
+Dans la nouvelle configuration du repo, copiez l'URL d'origin du repo et ajoutez-la à votre projet git avec cette commande :
+
+```bash
+$ git remote add origin https://github.com/<your username>/taskbox.git
+```
+
+Et pour finir pousser vos modifications sur le repo GitHub
+
+```bash
+$ git push -u origin master
+```
+
+### Netlify
+
+Netlify possède un service de déploiement continu intégré qui nous permettra de déployer storybook sans avoir à configurer notre propre CI.
+
+<div class="aside">
+Si vous utilisez CI dans votre entreprise, ajoutez un script de déploiement à votre configuration qui télécharge <code>storybook-static</code> vers un service d'hébergement statique comme S3.
+</div>
+
+[Créer un compte sur Netlify](https://app.netlify.com/start) et cloquez sur "créer un site".
+
+![Créer un site netlify](/intro-to-storybook/netlify-create-site.png)
+
+Ensuite, cliquez sur le bouton GitHub pour connecter Netlify sur GitHub. Cela lui permet d'accéder à notre repo taskbox.
+
+Maintenant, sélectionnez le repo GitHub taskbox à partir de la liste des options.
+
+![Connectez Netlify à votre repo](/intro-to-storybook/netlify-account-picker.png)
+
+Configurez Netlify en surlignant la commande de build à construire dans son CI et le répertoire dans lequel le site statique est généré. Pour la branche, choisissez `master`. Le répertoire est `storybook-static`. La commande de build utilise `yarn build-storybook`.
+
+![Paramètres Netlify](/intro-to-storybook/netlify-settings.png)
+
+<div class="aside"><p>Si votre déploiement échoue avec Netlify, ajoutez l'option <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a>  à votre script <code>build-storybook</code>.</p></div>
+
+Soumettez le formulaire pour construire et déployer le code de la branche `master` du repo taskbox.
+
+Une fois terminé, nous verrons un message de confirmation sur Netlify avec un lien vers le Storybook taskbox en ligne. Si vous suivez le tutoriel, votre Storybook déployé devrait être en ligne [comme cela](https://clever-banach-415c03.netlify.com/).
+
```

Comment:
@jeanphibaconnais when you're able can you adjust the link "come cela" to point at  https://clever-banach-415c03.netlify.app/ ?

Reply:
it's ok for this one.

Reply:
Thanks once more for updating it